package.json

```diff
@@ -13,9 +13,9 @@
     "postinstall": "manypkg check",
     "clean:packages": "yarn workspaces run clean",
     "build": "tsc -b ./tsconfig.monorepo.json",
-    "test": "jest",
-    "test:core": "npm test --prefix packages/core",
-    "dev": "node ./scripts/dev.js",
```

Comment:
this is equivalent and purely stylistic (up to personal preferences) - I would in general try to avoid such changes in PRs as they introduce some noise and don't bring any value

Reply:
Done 👍